### DISCUSSION TITLE
Localization problems

### DISCUSSION THREAD

**Discussion by kotleni** [y9vad9/kotlin-course#84] [General]
**I created this discussion to track localization issues for the course.**

**Comment by kotleni** [comment]
**Where:**
Kotlin -> Intro

**Language:**
UK

**Correction proposal:**
Replace "постараюсь" with "спробую"

**Proof:**
![telegram-cloud-photo-size-2-5472313300822900300-y](https://user-images.githubusercontent.com/38311102/203420964-343e1420-d6d4-44c0-a310-fc198e5ab019.jpg)

**Reply by y9vad9** [reply]
isn't it changing the sense? I don't think that 'постараюсь' is wrong variant.

**Reply by kotleni** [reply]
> isn't it changing the sense? I don't think that 'постараюсь' is wrong variant.

Okay dude. :)
